src/components/EventList/components/EventItem/index.jsx

```diff
@@ -1,10 +1,14 @@
-import { CardContent, Collapse, Grid, Typography } from '@mui/material';
+import { Button, CardContent, Collapse, Grid, Typography } from '@mui/material';
 import { useState } from 'react';
 import styles from './EventItem.module.css';
+import { AddToCalendarButton } from 'add-to-calendar-button-react';
 
 const EventItem = ({ event }) => {
   const [expanded, setExpanded] = useState(false);
 
+  
+
+  
```

Comment:
I don't think this extra whitespace is necessary. Generally we try to limit it to one new line if any.